### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by alexcuesta**
I removed an extra "0" in foss.html

### CODE REVIEW

**Review by anyulled — APPROVED** [review]

**Comment by anyulled** [comment]
great! you can merge this one now and create another to `gh-pages` to have it deployed in production

**Comment by alexcuesta** [comment]
Please, have a look at the entry column size. I had to change it from 6 to 5 because it did not fit the page width size.
I don't know much about Bootstrap but that made it work!

**Review by anyulled — APPROVED** [review]
Approved, tira pa' lante

**Comment by abelsromero** [comment]
'main' is publication now, no need for extras https://github.com/barcelonajug/jbcnconf_web/pull/1175#issuecomment-1142584723

**Comment by abelsromero** [comment]
> Please, have a look at the entry column size. I had to change it from 6 to 5 because it did not fit the page width size.

Can you provide an example? With md-5 I see a blank space on the left side

image

Iiuc :man_shrugging: :question:  Bootsrap's Grid splits the width in 12 columns, so we have 2 without spacing, it should be 6 to occupy the whole width. Then if you want to adjust based on resolution you can use the `xs`, `md` and `lg` this table sumarizes it https://getbootstrap.com/docs/3.3/css/#grid-options (beware not to google or it will point you to the latest version). We could do something like col-xs-12 col-sm-6 col-md-4 col-lg-3` to display 1 per row, 2, 3 or 4 depending on the width.

Another thing I found is that there's another grid in line 175 that messes with other things, for instance text does not extend to whole width. Maybe the real issue is that grid?

File: 2022/foss.html [hunk 1]
```diff
@@ -188,7 +188,7 @@ <h1>FOSS & Diversity Communities</h1>
                                         <div class="row alternative-row">
                                             <div class="col-lg-12 col-sm-12 col-xs-12 site-content-left">
                                                 {% for entry in site.data.communities %}
-                                            <div class="entry entry-{{ entry.company.level }} col-md-6">
+                                            <div class="entry entry-{{ entry.company.level }} col-md-5">
```
Comment (abelsromero) [inline 2022/foss.html:191]: `level` property does not exist in `communities.yml`, seems save to delete

**Comment by abelsromero** [comment]
If no one minds, I'll merge in a couple of houres and see if my theory makes sense from main branch.

**Review by anyulled — APPROVED** [review]

**Comment by alexcuesta** [comment]
Sorry I was planning to have a look tonight. I had not had a chance yet

**Comment by abelsromero** [comment]
> Sorry I was planning to have a look tonight. I had not had a chance yet

No problem, we can wait then, I did not mean to add any preassure.

**Comment by alexcuesta** [comment]
@abelsromero 

I think I have found the missing closing divs and fixed the mess.
I have also kept the previous column size.

Sorry I could not do this earlier. I had a very busy week.
Let me know if it is ok this time

File: 2022/foss.html (hunk 1 above)
Comment (alexcuesta) [reply 2022/foss.html]: You are right. Deleted that css style.

**Comment by abelsromero** [comment]
excelent :rocket:

**Pull request merged**